README.md

```diff
@@ -53,10 +53,13 @@ For questions, bug reports, and feature requests,
 please [file an issue](https://github.com/kylebgorman/wikipron/issues).
 
 If you would like to contribute to the `wikipron` codebase,
-please see [`CONTRIBUTING.md`](CONTRIBUTING.md).
+please see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 We keep track of notable changes in [CHANGELOG.md](CHANGELOG.md).
 
 ## License
 
-Apache 2.0. Please see [`LICENSE.txt`](LICENSE.txt) for details.
+Apache 2.0. Please see [LICENSE.txt](LICENSE.txt) for details.
+
+Please note that Wiktionary data has
```

Comment:
Slight emendation. Add: 

", as does the other data in the [languages](languages/) subdirectory."

to the end of this.

Reply:
Good call - I forgot about that subdirectory. Note added at d6e9b70.